### PR TITLE
СУХАРИКИ

### PR DESCRIPTION
**Pull request by tema5002**
it should be Collector not Collecter your bot is unusable update it now or i will physically explode

### CODE REVIEW

**Comment by Aflyde** [comment]
you must continue eating sand instead of commenting

**Comment by rech2020** [comment]
*eats sand*

**Comment by tema5002** [comment]
geometry dash is better

**Comment by rech2020** [comment]
polyhedra

**Comment by rotatable-cube** [comment]
the sun destroys badly written apples

**Comment by tema5002** [comment]
ААХХАХАХАХАХХАХАХХАХ ЭТО ЧТО ЗА ЛЯГУШКА

**Comment by tema5002** [comment]
also i bet all my life savings milenakos wants me to explode

**Comment by tema5002** [comment]
sadly i dont actually have any but whatever

File: config/aches.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "extrovert": {"title": "Extrovert", "description": "finish a trade with someone", "is_hidden": false, "category": "Cat Hunt"},
   "fast_catcher": {"title": "Fast Catcher", "description": "catch a cat in under 5 seconds", "is_hidden": false, "category": "Cat Hunt"},
   "slow_catcher": {"title": "Slow Catcher", "description": "catch a cat in over a hour", "is_hidden": false, "category": "Cat Hunt"},
-  "collecter": {"title": "Collecter", "description": "get all cat types", "is_hidden": true, "category": "Cat Hunt"},
+  "collector": {"title": "Collector", "description": "get all cat types", "is_hidden": true, "category": "Cat Hunt"},
```
Comment (milenakos) [inline config/aches.json:11]: changing id removes it from every single existing user not to mention requires a db migration

**Comment by milenakos** [comment]
therefore

**Pull request closed without merging**